packages/wonder-blocks-dropdown/src/components/action-item.tsx

```diff
@@ -223,7 +223,6 @@ const styles = StyleSheet.create({
     },
     shared: {
         minHeight: DROPDOWN_ITEM_HEIGHT,
-        height: DROPDOWN_ITEM_HEIGHT,
```

Comment:
Now I remember why I added this..... according to the Design spec, single-line items are expected to have a `40px` height, and in this case these would be `44px`. I would recommend looking at `OptionItem` to see how this is approached there (I noticed that those items remain at 40px on single line).

Reply:
Good catch! OptionItem uses 10px for vertical padding. Updated ActionItem to do the same using our new sizing tokens :) 
https://github.com/Khan/wonder-blocks/blob/b592ba98dc35a16e4b36d7f1af8a7fcab67d4f7d/packages/wonder-blocks-dropdown/src/components/option-item.tsx#L478-L479